database/migrations/2023_11_14_130544_add_type_column_to_nfts_table.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+use App\Enums\TokenType;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('nfts', function (Blueprint $table) {
+            $table->string('type')->default(TokenType::Erc721->value);
+        });
+
+        DB::table('nfts')->update([
+            'type' => TokenType::Erc721->value,
```

Comment:
did this work fine when storing erc1155 tokens? I don't know the exact differences from the API responses when fetching them, only that they differ in nature. If it's stored fine without weird overwrites (E.g. if multiple users own the same kind of ERC1155, how will it handle that as we assume 1 owner at the moment) then we can keep this, otherwise I'd suggest we leave it until we add proper ERC1155 support

Reply:
an example https://rarible.com/token/0x488130618bf46cd904fdfe1e2f94d377d73f5eb9:1 , only 1 token but there are 311 owners (22 remaining unowned). How would that be handled if we have address A that owns it and connects to dashbrd, but address B that also owns that NFT which connects at a later time?

Reply:
interesting, i might have overlooked the fact that erc1155 can have multiple owners, and we'd need to map nft to multiple wallets

Reply:
@crnkovic For multiple owners, see this task https://app.clickup.com/t/86dqnjq6h but that's for a follow up